algorithms/kernel/dtrees/forest/regression/inner/df_regression_train_container_v1.h

```diff
@@ -0,0 +1,93 @@
+/* file: df_regression_train_container_v1.h */
+/*******************************************************************************
+* Copyright 2014-2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+/*
+//++
+//  Implementation of decision forest container.
+//--
+*/
+
+#ifndef __DF_REGRESSION_TRAIN_CONTAINER_V1_H__
+#define __DF_REGRESSION_TRAIN_CONTAINER_V1_H__
+
+#include "algorithms/kernel/kernel.h"
+#include "algorithms/decision_forest/decision_forest_regression_training_types.h"
+#include "algorithms/decision_forest/decision_forest_regression_training_batch.h"
+#include "algorithms/kernel/dtrees/forest/regression/df_regression_train_kernel.h"
+#include "algorithms/kernel/dtrees/forest/regression/df_regression_model_impl.h"
+#include "service/kernel/service_algo_utils.h"
+
+namespace daal
+{
+namespace algorithms
+{
+namespace decision_forest
+{
+namespace regression
+{
+namespace training
+{
+namespace interface1
+{
+template <typename algorithmFPType, Method method, CpuType cpu>
+BatchContainer<algorithmFPType, method, cpu>::BatchContainer(daal::services::Environment::env * daalEnv)
+{
+    __DAAL_INITIALIZE_KERNELS(internal::RegressionTrainBatchKernel, algorithmFPType, method);
+}
+
+template <typename algorithmFPType, Method method, CpuType cpu>
+BatchContainer<algorithmFPType, method, cpu>::~BatchContainer()
+{
+    __DAAL_DEINITIALIZE_KERNELS();
+}
+
+template <typename algorithmFPType, Method method, CpuType cpu>
+services::Status BatchContainer<algorithmFPType, method, cpu>::compute()
+{
+    Input * input   = static_cast<Input *>(_in);
+    Result * result = static_cast<Result *>(_res);
+
+    const NumericTable * x = input->get(data).get();
+    const NumericTable * y = input->get(dependentVariable).get();
+
+    decision_forest::regression::Model * m = result->get(model).get();
+
+    const decision_forest::regression::training::interface1::Parameter * par =
```

Comment:
See comment in algorithms/kernel/dtrees/forest/classification/inner/df_classification_train_container_v2.h

Reply:
Fixed